api/src/main/java/org/openmrs/scheduler/db/SchedulerDAO.java

```diff
@@ -77,7 +77,6 @@
 	 * @param schedule to be created
 	 * @throws DAOException
 	 */
-	//public void createSchedule(Schedule schedule) throws DAOException;
```

Comment:
@Nuwantha remove the javadoc that did belong to the commented out method signature. this has to be done for every occurence. in this class and everywhere else.